rotkehlchen/premium/premium.py

```diff
@@ -822,6 +835,13 @@ def fetch_limits(self) -> UserLimits:
         log.debug(f'Fetched user limits from server: {self._cached_limits}')
         return self._cached_limits
 
+    def get_features(self) -> dict[str, bool]:
+        limits = self.fetch_limits()
+        return {
+            feature_label: limits.get(feature_label, False)  # type: ignore # default to False in case we deprecate the key. Ignore type because python returns builtins.object instaed of bool
```

Comment:
Also add link to the issue here

```suggestion
            feature_label: limits.get(feature_label, False)  # type: ignore # default to False in case we deprecate the key. Ignore type because mypy sees builtins.object instead of bool
```